src/main/java/ru/skypro/Main.java

```diff
@@ -5,101 +5,92 @@ public static void main(String[] args) {
 
         //number 1
         System.out.println("Задание 1");
-        int cashInMonth = 15000;
-        int capital = 2_459_000;
-        int conservation = 0;
-        int a = 0;
-        while (conservation < capital) {
-            System.out.println("Месяц " + a + " сумма накоплений равна " + conservation + " рублей.");
-            a++;
-            conservation = conservation + cashInMonth;
-            conservation = conservation + conservation / 100;
-        }
+        int[] arr = new int[3];
+        arr[0] = 1;
+        arr[1] = 2;
+        arr[2] = 3;
+
+        double[] arr2 = new double[3];
+        arr2[0] = 1.57;
+        arr2[1] = 7.654;
+        arr2[2] = 9.986;
 
+        int[] arr3 = {5, 7, 9, 11, 12};
+        System.out.println();
 
         //number 2
-        System.out.println("Задание 2");
-        int i = 1;
-        while (i <= 10) {
-            System.out.print(" " + i);
-            i++;
+        System.out.print("Задание 2");
+        System.out.println();
+        for (int i = 0; i < arr.length; i++) {
+            if (i == arr.length - 1) {
+                System.out.print(arr[i]);
+                break;
+            } else {
+                System.out.print(arr[i] + ", ");
+            }
```

Comment:
т.к. и в if, и в else есть одно и то же - вывод элемента - то лучше это сделать до if, а внутри if по условию, что если элемент не последний, то выводим ещё запятую, код был бы короче и понятнее
```suggestion
            System.out.print(arr[i]);
            if (i < arr.length - 1) {
                System.out.print(",");
            }
```